### ISSUE TITLE
Server doesn't suppor

### ISSUE THREAD

**Issue by KAMIKAZEUA** [strawberrymusicplayer/strawberry#963]
**Describe the bug**

**To Reproduce**
Play http://listen.trancebase.fm/tunein-mp3-pls then pause, then play using F8 key after some time.

**Expected behavior**

**Screenshots:**

**System Information:**
 - Operating system:
 - Strawberry Version:

**Additional context**

**Comment by jonaski** [comment]
Duplicate of #846

**Comment by jonaski** [comment]
No, it won't be "fixed", it's gstreamer limitation, ie. gstreamer soup is not reading the http headers.

**Comment by KAMIKAZEUA** [comment]
@jonaski  I'd love to, but I'm ObjC only), however sounds like a 2 lines fix.. too bad that it's occurs so often but can't be fixed quickly.

**Comment by jonaski** [comment]
You can't pause a live stream, I don't see the big deal of just stopping and starting the playback when you are playing a radio stream.
I'd rather see it solved at a lower level in gstreamer when that's fully possible than creating a "fix" in Strawberry which will be more of a workaround. I don't have the time to dig into it myself.

**Comment by KAMIKAZEUA** [comment]
Well, if logically you can't pause livestream for sure, so instead of pause -> just stop.. simple solution?
